pytrevl/charts.py

```diff
@@ -280,6 +281,15 @@ def __init__(self, query, y, name, inner='30%', outer='60%', **kwargs):
             **kwargs,
         )
 
+class CustomChart (BaseChart):
+    def __init__(self, trevl_code):
```

Comment:
We need to extract the `id` at least otherwise the `str` represenation is broken:
```suggestion
    def __init__(self, trevl_code):
        self.id = trevl_code['id']
```